ddtrace/sampler.py

```diff
@@ -304,44 +341,33 @@ def sample(self, span):
         :rtype: :obj:`bool`
         """
         # If there are rules defined, then iterate through them and find one that wants to sample
-        matching_rule = None  # type: Optional[SamplingRule]
+        sampler = None  # type: Optional[Union[SamplingRule, RateLimiter]]
 
         # Go through all rules and grab the first one that matched
         # DEV: This means rules should be ordered by the user from most specific to least specific
```

Comment:
I know this isn't your change, but do we state this anywhere in the docs? I couldn't find it here. https://docs.datadoghq.com/tracing/trace_ingestion/mechanisms/?tab=environmentvariables#in-tracing-libraries-user-defined-rules

Reply:
we don't state it in our API docs either: https://ddtrace.readthedocs.io/en/stable/configuration.html

if you wanted to open PRs for both of those would be 🙌🏻